portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACNavigation.path

```diff
@@ -27,6 +27,21 @@
 	<td></td>
 </tr>
 <!-- TABS/OPTIONS AVAILABLE IN DIFFERENT PLACES E.G: ASSETS, SEGMENTS, SITES -->
+<tr>
+	<td>DEFINITIONS_ATRIBUTES_TABS</td>
+	<td>//ul[contains(@class,'nav-root')]//b[text()='${key_tabName}']</td>
+	<td></td>
+</tr>
```

Comment:
Hi @91nilo, This path can work the same way as `TAB_NAME`, just make a simple change to the existing path.

Please change the path of the `TAB_NAME` to `//ul[contains(@class,'nav-root')]//*[text()='${key_tabName}']`, that way we won't have two paths basically doing the same thing and with this change you can use the `switchTab` macro